.github/workflows/lint.yml

```
@@ -0,0 +1,66 @@
+    branches: [ "main", "develop" ]
+  pull_request:
+    branches: [ "main", "develop" ]
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:8.0
+        container_name: safetygps-mysql
```

actionlint (1.7.8)

unexpected key "container_name" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)